internal/test/e2e/e2e_test.go

```diff
@@ -83,8 +85,15 @@ func getSpec(t *testing.T, o interface{}) interface{} {
 
 // Starts a Kind cluster on the local machine, exposing port 2379 accepting ETCD connections.
 func startKind(t *testing.T, ctx context.Context) (*cluster.Context, error) {
-	t.Log("Starting Kind cluster")
-	kind := cluster.NewContext("etcd-e2e")
+	clusters, err := cluster.List()
+	for _, c := range clusters {
+		if c.Name() == *fKindClusterName {
+			t.Log("Found existing Kind cluster")
```

Comment:
are we confident that reusing a cluster will not have any side effects that would cause tests to run inconsistently?

Reply:
There are some possible inconsistencies since re-applying the sample cluster manifest won't always change the existing cluster and the operator won't have to bootstap it, but these will be revealed when the tests get run in a fresh environment by CI.  

In #69 I've modified things so that every group of tests gets run in a fresh namespace. That should make the test results more reproducable when re-using an existing cluster.